draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -94,34 +96,45 @@ The following diagram depicts the conceptual interactions.
                      +------->|               |<------------------------>|               |
   (1) Generate       |        |    Client     |    (6) Interaction       | Authorization |
   Attestation Key    |        |   Instance    |    using Client          |    Server     |
-                     +--------|               |    Key Attestation + PoP |               |
+                     +--------|               |    Attestation + PoP     |               |
                               |               |    for authentication    |               |
                               +---------------+                          +---------------+
                                  / \      |
                                   |       |
                                   +-------+
                                 (5) Generate
-                              Key Attestation PoP
+                           Client Attestation PoP
 ~~~
 
-Note defining the protocol for steps 2 and 4, including how the client instance authenticates with the client backend is out of scope of this specification.
+Note that the communciation protocols used for requesting and sending a Client Attestation JWT between the client instance and its backend (steps 2 and 4) and the data how the client instance authenticates with the client backend is out of scope of this specification.
```

Comment:
This text feels a bit more complex to me? I think the previous text was a bit clearer in this explanation?